packages/bpk-component-image/src/withLazyLoading.js

```diff
@@ -57,8 +58,10 @@ export default function withLazyLoading(Component, document) {
     }
 
     removeEventListeners() {
-      document.removeEventListener('scroll', this.checkInView);
+      const passiveArgs = this.supportsPassiveEvents() ? { passive: true } : {};
+      document.removeEventListener('scroll', this.checkInView, { capture: true, ...passiveArgs });
```

Comment:
When removing an event listener, do you have to provide the options? What happens if you omit them?